tests/functional/transaction_timeout.ts

```diff
@@ -21,7 +21,8 @@ const blockchainSettings = {
   BATCH_CREATION_INTERVAL_IN_SECONDS: 5,
   READ_DIRECTORY_INTERVAL_IN_SECONDS: 5,
   UPLOAD_CLAIM_INTERVAL_IN_SECONDS: 5,
-  TRANSACTION_MAX_AGE_IN_SECONDS: 60,
+  MAX_BLOCK_HEIGHT_DELTA: 0,
```

Comment:
I feel like we should test this with some number greater than 0.

0 doesn't really test whether the math is correct.

Reply:
I'll set it to 1 and mine 2 then.